superset/views/core.py

```diff
@@ -2430,14 +2432,30 @@ def results(self, key):
     @has_access_api
     @expose("/stop_query/", methods=["POST"])
     @event_logger.log_this
+    @backoff.on_exception(
+        backoff.constant,
+        DatabaseError,
```

Comment:
This will now throw and return a 500 when there's an exception other than a DatabaseError, correct? I think that is an improvement to the generic Exception catching before, but wanted to make sure

Reply:
Yes.